Extractor/UAExtractor.cs

```diff
@@ -287,7 +299,17 @@ private async Task RunExtractorInternal(int startTimeout = -1)
             }
 
             Started = true;
-            startTime.Set(new DateTimeOffset(DateTime.UtcNow).ToUnixTimeMilliseconds());
+
+            if (
+                serverSubscription is not null
+                && Config.Extraction.RebrowseTriggers is not null
```

Comment:
One thing we do quite a lot is to just check if the manager type is null, because we only create it if we are going to need it. Since we check `config.Extraction.RebrowseTriggers` when creating the manager, we don't need to check it here.

Reply:
I also checked the `serverSubscription` because the compiler throws this warning `Dereference of a possibly null reference. [Extractor]` without it.

Reply:
I mean, check only serverSubscription. For the nullability warning, you can ignore it here. Use the `!` operator, if you need to.